sos/report/plugins/foreman_installer.py

```diff
@@ -88,10 +88,10 @@ class RedHatForemanInstaller(ForemanInstaller, RedHatPlugin):
     def setup(self):
 
         self.add_file_tags({
-            '/var/log/foreman-installer/satellite.log.*':
-                ['insights_satellite_log' 'satellite_installer_log'],
-            '/var/log/foreman-installer/capsule.log.*':
-                ['insights_capsule_log' 'capsule_installer_log'],
```

Comment:
Would it be better to make this `(.*)?` so that we also tag rotated logs, or is that not relevant?

Reply:
So the current spec is only the first file, it isn't setup for glob collection of all the files. So I removed the .* for any tag that had that at the end but insights didn't collect via glob all the files.

Reply:
Ack, no need to tag rotated files then if the tag won't be useful on them.